src/main/webapp/index.html

```diff
@@ -94,20 +94,18 @@ <h3 class="panel__section-heading">Tasks</h3>
             </div>
             <div class="panel__content-entry">
               <img class="panel__content-icon" alt="clipboard-icon" src="assets/clipboard.png">
-              <p class="panel__text-large" id="tasks-due-today">4 due today
+              <p class="panel__text-large"><span id="tasks-due-today">4</span> due today
```

Comment:
Consider replacing the placeholders in the HTML as well. Otherwise, I think there will be a period of time when the hard-coded values are shown, probably while the page is authenticating.

Reply:
You're absolutely right - fixed!